pandas/core/dtypes/dtypes.py

```diff
@@ -126,28 +130,28 @@ class PandasExtensionDtype(ExtensionDtype):
     isnative = 0
     _cache = {}  # type: Dict[str_type, 'PandasExtensionDtype']
 
-    def __str__(self):
+    def __str__(self) -> str_type:
```

Comment:
@WillAyd on your how-to-type-annotate can you add when / why to use ``str_type``